app/core/base.js

```diff
@@ -159,4 +161,15 @@ FauxtonAPI.setSession = function (newSession) {
 
 FauxtonAPI.reducers = [];
 
+FauxtonAPI.json = (url, opts = {}) => fetch(
```

Comment:
Could you move this into its own file, maybe something like `ajax`.  This PR https://github.com/apache/couchdb-fauxton/pull/745 had a good idea. Unfortunately it doesn't look like they will complete it. Could you rather do something similar like that for all our ajax requests with fetch